src/domain/Features.ts

```diff
@@ -1,6 +1,6 @@
 import getApiClient from '../service/getApiClient';
 
-export const FEATURES = ['chat', 'video', 'call_recording', 'fax', 'mobile_double_call', 'mobile_gsm', 'meeting'];
+export const FEATURES = ['chat', 'video', 'call_recording', 'fax', 'mobile_double_call', 'mobile_gsm', 'meeting', 'caller_id'];
```

Comment:
Beware that this file does not handle feature flag. Features [are available by default](https://github.com/wazo-platform/wazo-js-sdk/blob/master/src/domain/Features.ts#L88).